lms/djangoapps/discussion/templates/discussion/discussion_mfe_embed_js.template

```diff
@@ -0,0 +1,35 @@
+## mako
+
+<%!
+
+from openedx.core.djangolib.js_utils import dump_js_escaped_json, js_escaped_string
+from cms.djangoapps.contentstore import utils
+%>
+
+const domain = (new URL("${discussions_mfe_url}"));
+const course_id = "${course_key}"; 
+let current_pathname = window.location.pathname.slice();
+window.addEventListener("message", (event) => {
+    if ((event.origin !== domain.origin) || (event.data.type !== "discussions-mfe-tab-embed"))
+        return;
+    const {postId ,topicId} = event.data.payload;
+    if (topicId){
+        let new_url = "".concat(
+            current_pathname.split('discussion/forum/')[0],
+            "discussion/forum/",
+            topicId
+            )
+        if (postId){
+            new_url = new_url.concat("/threads/",postId);
+        }
+        window.history.pushState(null,null , new_url);
+    }
+    else{
+        let new_url = "".concat(
+            current_pathname.split('discussion/forum/')[0],
+            "discussion/forum/",
+            )
+        window.history.pushState(null,null , new_url);
+    }
+
+}, false);
```

Comment:
Don't use a template for this. Just use a static JavaScript file that can load the needed values from the page. Or, since it's small enough, just include this in the main HTML. 

Also, please format the code based on the [edX style guide](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/javascript-guidelines.html).

Reply:
I included it in the HTML template.